lib/classes/Database.js

```diff
@@ -1,10 +1,10 @@
-const {Sequelize} = require("sequelize");
+const { Sequelize } = require("sequelize");
 const Guild = require("../database/models/Guild.js");
 const GuildChannel = require("../database/models/GuildChannel.js");
 const GuildMember = require("../database/models/GuildMember.js");
 const User = require("../database/models/User.js");
-const fs = require("fs").promises;
-const {log} = require("../classes/Utils.js");
+const { log } = require("../classes/Utils.js");
+const DbConfig = require("../../config/db.json");
```

Comment:
Not sure about this, becuase require returns just object, not a class. I think it's need defferent name like `dbConfig`.